core/sql/optimizer/Inlining.cpp

```diff
@@ -618,10 +618,14 @@ RETDesc *GenericUpdate::createOldAndNewCorrelationNames(BindWA *bindWA, NABoolea
     rd = new (bindWA->wHeap()) RETDesc(bindWA);
   }
 
+  /*  if ((getOperatorType() != REL_UNARY_INSERT) || 
+      getUpdateCKorUniqueIndexKey() ||
+      ((getOperatorType() == REL_UNARY_INSERT) &&((Insert *)this)->isMerge()) ||
+      ((getOperatorType() == REL_UNARY_INSERT) && ((Insert *)this)->isUpsert() && (CmpCommon::getDefault(TRAF_UPSERT_TO_EFF_TREE) == DF_ON )))  */
```

Comment:
It's better to not leave this old code here... it just clutters things up and makes reading the code more confusing. We can always recover the old code from the repository if needed.